tools/topology/topology2/cavs-nocodec.conf

```diff
@@ -620,6 +627,30 @@ Object.Pipeline.io-gateway-capture [
 
 IncludeByKey.PASSTHROUGH {
 "false" {
+		Object.Pipeline.host-gateway-capture [
+			{
+				index 	11
+
+				Object.Widget.copier.1 {
+					stream_name 'SSP2 Capture'
+
+					<include/components/src_passthrough_format.conf>
+				}
+			}
+		]
+
+		Object.Pipeline.src-capture [
```

Comment:
You can simply replace this with Object.Widget.src.1 {} directly and remove src-capture.conf